win32/build/confutils.js

```diff
@@ -3067,7 +3067,7 @@ function toolset_get_compiler_version()
 		var command = 'cmd /c ""' + PHP_CL + '" -v"';
 		var full = execute(command + '" 2>&1"');
 
-		if (full.match(/clang version ([\d\.]+) \((.*)\)/)) {
+		if (full.match(/clang version ([\d\.]+)(?: \((.*)\))?/)) {
```

Comment:
Do we need this group at all? We're not actually using it.

Reply:
No, probably not. I don't know what this was capturing, it doesn't match anything in modern versions of clang-cl.

Reply:
clang 9.0.0 says:
````
clang version 9.0.0 (tags/RELEASE_900/final)`
````

So the ` \((.*)\)` was likely meant to catch the info in parens, although it is indeed not used, and as such could be completely removed. And actually, even the version is not used (well, it could be used in config.w32, but likely almost anybody assumes that these are for VS).

For what it's worth, I've just manage to do a successful minimal build (`--with-toolset=clang --disable-all --enable-cli`) of PHP-8.0 with clang 9.0.0, but PHP-8.1; I'll check what a recent version of clang produces.